src/utils/proxy.js

```diff
@@ -378,15 +367,36 @@ const startProxy = async function (settings, addonsUrls, configPath, projectDir)
     }
 
     proxy.web(req, res, options)
+  }
+
+  return onRequest
+}
+
+const createSecureServer = ({ onRequest, cert, key }) => https.createServer({ cert, key }, onRequest)
```

Comment:
[sand] Would it be simpler to inline that function?

Reply:
Done in https://github.com/netlify/cli/pull/2132/commits/e763e9c33e23f3cdbee88b4f827d5265a372210e